server/models/handlers/availability.js

```diff
@@ -1,29 +1,22 @@
 // Gets the time availabilities for a specific worker for a specific school.
-async function getWorkerAvailability(workerId, schoolId, startTime, endTime) {
-    const sampleAvailableTimes = [
-        {
-            workerTimeslotId: 1,
-            startTime: '08:00:00', 
-            endTime: '08:30:00',
-            date: '2020-10-20'
-        },
-        {
-            workerTimeslotId: 2,
-            startTime: '08:30:00', 
-            endTime: '09:00:00',
-            date: '2020-10-20'
-        },
-        {
-            workerTimeslotId: 3,
-            startTime: '09:00:00', 
-            endTime: '09:30:00',
-            date: '2020-10-20'
-        }
-    ];
+const _ = require('lodash');
+const availabilityModel = require('../db/availability');
+//need userTypes or Workerstatus?
+
+async function getAvailabilityDetails(workerId, schoolId, startTime, endTime) {
+
+    // Worker credentials do not need to be checked as the getAvailabilityDetails method will return an error if the worker credentials are invalid.
+
```

Comment:
Hi Nancy,
I would recommend removing the blank lines around this comment to reduce unnecessarily long code files.

Reply:
Hi Amy,
Good catch!
My action: I've removed the blank lines on line 7 and 9 of the server/models/handlers/availability.js and used right click> format document to ensure the file adheres to the required formatting standards.

Please confirm if this takes care of your suggestion!